jwql/utils/preview_image.py

```diff
@@ -161,6 +161,14 @@ def find_limits(self, data):
         # Ignore any pixels that are NaN
         finite = np.isfinite(data)
 
+        # If all non-science pixels are NaN then we're sunk. Scale
+        # from 0 to 1.
+        if not np.any(finite):
```

Comment:
Are non-science pixels always `NaN`?

Reply:
No. Avoiding NaN pixels is a separate issue. In this block I'm looking across all pixels (not just non-science pixels) for NaN values. I don't expect this condition where all pixels are NaN to ever happen, but I figured it would be good to cover it just in case my expectations are wrong.